pyboy/plugins/game_wrapper_metroid_ii.py

```diff
@@ -0,0 +1,191 @@
+          3  | 383 383 383 321 322 383 383 383 383 383 383 383 383 383 383 383 383 383 300 294
+          4  | 383 383 383 383 323 290 291 383 383 383 313 312 311 318 319 320 383 290 291 383
+          5  | 383 383 383 383 324 383 383 383 383 315 314 383 383 383 383 321 322 383 383 383
+          6  | 383 383 383 383 324 293 292 383 383 316 383 383 383 383 383 383 323 383 383 383
+          7  | 383 383 383 383 324 383 383 298 383 317 383 383 383 383 383 383 324 383 383 383
+          8  | 319 320 383 383 324 383 383 297 383 317 383 383 383 152 140 383 324 383 383 307
+          9  | 383 321 322 383 324 294 295 296 383 325 383 383 383 383 383 383 326 272 274 309
+          10 | 383 383 323 383 326 383 383 383 2   18  383 330 331 331 331 331 331 331 331 331
+          11 | 274 383 324 272 274 272 274 272 274 272 274 334 328 328 328 328 328 328 328 328
+          12 | 331 331 331 331 331 331 331 331 331 331 331 328 328 328 328 328 328 328 328 328
+          13 | 328 328 328 277 278 328 328 328 328 328 328 328 328 277 278 328 328 277 278 277
+          14 | 328 277 278 279 281 277 278 328 328 277 278 277 278 279 281 277 278 279 281 279
+          15 | 278 279 281 280 282 279 281 277 278 279 281 279 281 280 282 279 281 280 282 280
+        ```
+
+        Returns
+        -------
+        memoryview:
+            Simplified 2-dimensional memoryview of the screen
+        """
+        return PyBoyGameWrapper.game_area(self)
+
+    def game_over(self):
+        return self._game_over
+
+    def __repr__(self):
+        # yapf: disable
+        return (
+            f"Metroid II:\n" +
+            # TODO add relevant variables to print for debugging purposes
+            f"XY Pixels: {(self.x_pos_pixels, self.y_pos_pixels)}\n" + 
+            f"XY Area: {(self.x_pos_area, self.y_pos_area)}\n" + 
+            f"Pose: {self.pose}\t" + 
+            f"Facing: {self.samus_facing}\n" + 
+            f"Upgrades: {self.current_major_upgrades}\n" + 
+            f"Water Info: {self.water_info}\n" + 
+            f"Curr Beam: {self.current_beam}\n" +
+            f"Curr E tanks: {self.current_e_tanks}\n" + 
+            f"Curr HP: {self.current_hp}\n" + 
+            f"Curr Full E Tanks: {self.current_full_e_tanks}\n" + 
+            f"Curr missiles: {self.current_missiles}\n" + 
+            f"Curr missile capacity: {self.current_missile_capacity}\n" + 
+            f"Displayed HP: {self.displayed_hp}\n" + 
+            f"Displayed Missiles: {self.displayed_missiles}\n"+
+            f"GMC: {self.global_metroid_count}\n" +  
+            f"LMC: {self.global_metroid_count}\n"
+            # I don't like seeing the huge grid whne trying to develop
+            # The sprite list can be messy too, since many things (especially
+            # samus) are composed of multiple sprites, so simply running causes
+            # the output to constantly change "shape", making it very difficult
+            # to read
+            # super().__repr__()
```

Comment:
Although I understand, please add it back in, so each game wrapper follows the same structure.